src/bin/pg_autoctl/cli_create_drop_node.c

```diff
@@ -549,6 +586,35 @@ cli_create_monitor(int argc, char **argv)
 	}
 
 	log_info("Monitor has been succesfully initialized.");
+
+	if (createAndRun)
+	{
+		char *channels[] = { "log", "state", NULL };
+
+		log_info("Contacting the monitor to LISTEN to its events.");
+		pgsql_listen(&(monitor.pgsql), channels);
+
+		/*
+		 * Main loop for notifications.
+		 */
+		for (;;)
+		{
+			if (!monitor_get_notifications(&monitor))
```

Comment:
any way to reuse/share the code from cli_service.c here?

Reply:
Yeah that's a good one, done now.